src/annotator/config/settings.js

```diff
@@ -102,9 +102,22 @@ function configFuncSettingsFrom(window_) {
   return window_.hypothesisConfig();
 }
 
+/**
+ * Return true if the client is from a browser extension.
+ *
+ * @returns {boolean} true if this instance of the Hypothesis client is one
+ *   distributed in a browser extension, false if it's one embedded in a
+ *   website.
+ *
+ */
+function isBrowserExtension(config) {
+  return !(config.app.startsWith('http://') || config.app.startsWith('https://'));
```

Comment:
`startsWith` is polyfilled in `src/shared/polyfills.js` which is loaded for both the sidebar and annotator bundles. It is worth being aware however that we do not load the polyfills in the boot bundle for size reasons, so if you're ever writing code which is used in that micro-bundle, it needs to stick to IE 10/11-safe code.